package.json

```diff
@@ -9,8 +9,12 @@
   "license": "MIT",
   "author": "simonihmig <simon.ihmig@kaliber5.de>",
   "scripts": {
```

Comment:
We have not previously been running prettier on this repo, so we have more changes than normal